src/components/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { Box, Button, Typography } from '@mui/material'
+
+const Dashboard = () => {
+  const routesNotAuthed = [{ path: '/', label: 'Register/Login page' }]
+  const routesAuthed = [
+    { path: '/callback', label: 'Auth Callback' },
```

Comment:
In my opinion, we don't need this route. It is used during registration. If we just click on this link, we will only see the logo